### PR TITLE
Blog - A Practical Guide to the Linux Uniq Command

### PR DESCRIPTION
**Pull request by msali123**
Done with allchecks.

### CODE REVIEW

**Comment by msali123** [comment]
@balapriyac [Preview Link](https://6470526912128c5073aa0d6c--ephemeral-narwhal-3ac3b2.netlify.app/blog/linux-uniq-command/)

File: blog/_posts/2023-06-26-linux-uniq-command.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+~~~
+
+As before, you're first using the `tail` command to display everything except the first line, then sorting the third column with the `sort` command.
+
+Then the [`sed`](https://earthly.dev/blog/sed-find-replace/) command is replacing all commas with a space character, and the `uniq` command can look for duplicates in the third column because you specified that you want to skip the first two fields when you're comparing values.
+
+### Implementing a Cleaner Output
+
+Suppose you only want to display the country names and their distinct value counts. You're going to need the help of the `[cut](https://www.tutorialspoint.com/unix_commands/cut.htm)` command. The `cut` command in Linux allows you to extract specific sections or columns of text from files or command output:
+
```
Comment (balapriyac) [inline blog/_posts/2023-06-26-linux-uniq-command.md:262]:
```suggestion
Suppose you only want to display the country names and their distinct value counts. You're going to need the help of the [`cut`](https://www.tutorialspoint.com/unix_commands/cut.htm) command. The `cut` command in Linux allows you to extract specific sections or columns of text from files or command output:

```

**Review by balapriyac — CHANGES_REQUESTED** [review]
@msali123 Just one small formatting edit.

**Comment by msali123** [comment]
@balapriyac thank, [changes done](https://647077912d38c16ced66c99d--ephemeral-narwhal-3ac3b2.netlify.app/blog/linux-uniq-command/)

**Review by jalletto — APPROVED** [review]

**Pull request merged**